src/app/tests/suites/certification/Test_TC_TCCM_1_1.yaml

```diff
@@ -75,7 +75,7 @@ tests:
       response:
           constraints:
               type: list
-              contains: [0, 1, 2, 3, 65528, 65529, 65531, 65532, 65533]
+              contains: [0, 1, 65528, 65529, 65531, 65532, 65533]
```

Comment:
Note that this is also part of https://github.com/project-chip/connectedhomeip/pull/29339, but that's blocked on unrelated issues... which is the problem with do-it-all-at-once PRs.